docs/usage.md

```diff
@@ -3,64 +3,63 @@
+    - **Recommendation** : Recommendation Tab contains following details:
+        - *PostgreSQL* : Database settings for improved performance.
+        - *PHP-FPM* :  PHP-FPM settings to boost processing speed.
         - *Workflow Engine* : Fine-tune worker configurations for better task execution and to enhance request handling.
         - *RabbitMQ* : Streamline message queuing to improve communication between services.
         - *SWAP SPACE* : Ensure adequate swap space to maintain stability under heavy load.
 # Configuration Recommendation
 How to read Configuration Recommendation Record
 - When you navigate to Configuration Recommendation Record's details view. There will be following tabs
-    - **Assessment** : Assessment Tab will contain following things:
+    - **Assessment** : Assessment Tab contains following details:
         - ***System Resources*** : System resources such as vCPUs, RAM, IOPS, and Disk Latency are crucial for determining the overall performance of an application or system.
-            - **vCPUs** represent the processing power allocated to a virtual machine, impacting the system's ability to handle concurrent tasks.
-            - **RAM** is vital for holding active data and instructions, influencing how efficiently applications run.
+            - **vCPUs** 
+            - **RAM** 
             - **IOPS (Input/Output Operations Per Second)** measures the speed of disk read/write operations, essential for data-intensive tasks.
-            - **Disk Latency** refers to the time delay in accessing data from storage, with lower latency improving responsiveness.
-    - **Recommendation** : Recommendation Tab will contain following things:
-        - *PostgreSQL* : Adjust database settings for improved performance.
-        - *PHP-FPM* : Adjust php-fpm settings to boost processing speed.
+            - **Disk Latency** refers to the time delay in accessing data from storage. Lower the latency greater the responsiveness.
+    - **Recommendation** : Recommendation Tab contains following details:
+        - *PostgreSQL* : Database settings for improved performance.
+        - *PHP-FPM* :  PHP-FPM settings to boost processing speed.
         - *Workflow Engine* : Fine-tune worker configurations for better task execution and to enhance request handling.
         - *RabbitMQ* : Streamline message queuing to improve communication between services.
         - *SWAP SPACE* : Ensure adequate swap space to maintain stability under heavy load.
 
-# Step to Apply Generated Configuration Recommendations
+# Step to Apply Recommended Configuration 
 Steps to incorporate the recommendations
-1. Login to Terminal of Your VM via SSH
-2. Following are the ways to implement above configuration either using cli command or by manually editing configuration file:
-    - Method 1 :
+1. Connect SSH session to your VM
```

Comment:
any pre-req? of backup instructions are needed?
is this applicable for FSR on-prem as well as on-cloud?